config/serverless.yml

```
@@ -0,0 +1,40 @@
+service: laravel
+
+provider:
+    name: aws
+    # The AWS region in which to deploy (us-east-1 is the default)
+    region: us-east-1
+    # The stage of the application, e.g. dev, production, staging… ('dev' is the default)
+    stage: dev
+    runtime: provided
+
+package:
+    # Directories to exclude from deployment
+    exclude:
+        - node_modules/**
+        - public/storage
+        - resources/assets/**
+        - storage/**
+        - tests/**
+
+functions:
+    # This function runs the Laravel website/API
+    web:
+        handler: public/index.php
+        timeout: 28 # in seconds (API Gateway has a timeout of 29 seconds)
+        layers:
+            - ${bref:layer.php-74-fpm}
+        events:
+            -   http: 'ANY /'
+            -   http: 'ANY /{proxy+}'
```

Would it make more sense to use HTTP API here?
Faster, cheaper and simpler as you probably know.

```suggestion
        events:
            -   httpApi: '*'
```

---

Indeed, I prefer using this too. However it has a major downside: it doesn't support HTTP -> HTTPS redirection.

That means that HTTP APIs are not really suitable for hosting websites, unless you add CloudFront (which is more complexity).

I want to move the whole Bref documentation to HTTP APIs eventually, but for now I am documenting using REST APIs so that "it works out of the box".